### PR TITLE
[TASK] Move to a clean ES6 source file and integrate the official logger

### PR DESCRIPTION
**Pull request by Inkdpixels**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,12 +34,13 @@
...
-    "@reduct/build-tools": "^1.4.5"
+    "@reduct/build-tools": "^1.5.2",
+    "@reduct/logger": "^1.1.0"
   },
   "reduct": {
     "sourceFolder": "Src/",
     "distributionFolder": "Dist/",
     "entryFile": "Registry.js",
-    "globalPackageName": "registry"
+    "globalPackageName": "reductRegistry"
```
Comment (akoenig) [inline package.json]: @Inkdpixels Can you clarify this a little bit? :) This would result in `window.reductRegistry` instead of our namespace `window.reduct.registry`.

`browserify` is cool with passing `reduct.registry` as string to the `standalone` attribute: https://github.com/reduct/build-tools/blob/1792bfe8bba3fc2ecfd9437a5b66d6595b598e2b/Src/Bin/Build.js#L21

Comment (Inkdpixels) [inline package.json]: Hey @akoenig - This is due to the fact that the UMD-Wrapper itself is now handled by browserify, since I couldn't export a clean browserify build through our UMD-Wrapper.

A clean browserify bundle will always return the following function if the `standalone`/`umd` option was _not_ enabled:

`function s(o,u)`  [link](https://github.com/reduct/registry/blob/task/moveToCleanES6ModuleStructure/Dist/Registry.js#L37)

This function always expects arguments which are internals of the bundle itself. Thus I had to enable the `standalone` option, and this option converts, as it is with any umd wrapper option out there, any dots in our global namespace into a camelCase name.

So basically the reduct object in the global namespace is now deprecated. and should only be initiated if necessary.

Comment (akoenig) [inline package.json]: `browserify` is capable of generating the respective namespace for us:

From the docs:

> When opts.standalone is a non-empty string, a standalone module is created with that name and a umd wrapper. You can use namespaces in the standalone global export using a . in the string name as a separator, for example 'A.B.C'. The global export will be sanitized and camel cased.
